src/lib-components/layout/DateFilter.vue

```diff
@@ -1,27 +1,27 @@
 <script setup lang="ts">
-import { DateRange } from "@/composables/date"
 import { ref } from "vue"
+import { DateRange } from "@/composables/date"
+import { SortDir } from "@/composables/list"
 import DateRangePicker from "../forms/DateRangePicker.vue"
 import Select from "@/lib-components/forms/Select.vue"
 
 const props = defineProps<{
   dateRange: DateRange
-  sortDir: string
-  title: string
```

Comment:
I'm okay with dropping this prop.  Looking at the Archive instance in the `ReviewsList` I don't know that it adds a ton of value, the context still appears clear without it.  @profsmallpine any concerns there?

Reply:
Yeah all good dropping this